ui/ts/components/navbar.ts

```diff
@@ -15,7 +15,7 @@ module Components {
   export module NavigationBar {
     import MithrilVirtualElement = _mithril.MithrilVirtualElement;
     export interface Target {
-      view: string | MithrilVirtualElement | MithrilVirtualElement[];
+      view: string | MithrilVirtualElement[];
```

Comment:
These changes appear to be unrelated; they appear to undo a previous improvement, actually.